### DISCUSSION TITLE
Using fluid-tailwind classes within @tailwindcss/typography config

### DISCUSSION THREAD

**Discussion by abredikis** [barvian/fluid-tailwind#34] [Help]
Hi,

I'm relatively new to Tailwind and its plugins, and I'm trying to use `fluid-tailwind` with the `@tailwindcss/typography` plugin. 
Specifically, I'd like to overwrite the default `font-size` of headings and make them responsive without the use of breakpoints.

Is it possible to achieve this when using `@tailwindcss/typography`? If using `@apply` isn't possible currently, are there any other ideas you can suggest (besides writing `clamp` functions)?

I've had no luck making it work with this config `tailwind.config.js`:

```
const config: Config = {
  content: {
    files: [
      './src/pages/**/*.{js,ts,jsx,tsx,mdx}',
      './src/components/**/*.{js,ts,jsx,tsx,mdx}',
      './src/app/**/*.{js,ts,jsx,tsx,mdx}',
    ],
    extract,
  },
  theme: {
    fontSize,
    screens,
    fluid: (({ theme }) => ({
      defaultScreens: [theme('screens.sm'), theme('screens.xxl')],
    })) satisfies FluidThemeConfig,
    extend: {
      typography: ((theme) => ({
        DEFAULT: {
          css: {
            h1: {
              '@apply ~text-xl/2xl': {},
            },
          },
        },
      })) satisfies ResolvableTo<Record<string, unknown>>,
    },
  },
  plugins: [fluid, typography],
};
```

Thanks in advance for your help!

**Comment by barvian** [comment]
Good question! This may be a Tailwind bug/limitation but I'm waiting to hear back on their Discord. I suppose you could use the `prose-` variants to accomplish this in the meantime, i.e.

```
<div class="prose prose-h1:~text-xl/2xl">
```
[Tailwind Play example](https://play.tailwindcss.com/K37O8iCtZY)

**Reply by firxworx** [reply]
Yep this is why I filed https://github.com/barvian/fluid-tailwind/issues/43 to throw it out there, its awesome to see you're looking into it and following up on their Discord, I just saw this comment today! 

Other questions like https://github.com/barvian/fluid-tailwind/discussions/40 are related

**Reply by barvian** [reply]
Yeah it seems like a Tailwind limitation, I often encounter issues when trying to `@apply` classes with modifiers from the config ([repro](https://play.tailwindcss.com/RidLrXHkOv?file=config)) 🫤
